libraries/bts_7200_current_sense/test/test_bts_7200_current_sense.c

```diff
@@ -0,0 +1,52 @@
+#include "bts_7200_current_sense.h"
+#include "delay.h"
+#include "interrupt.h"
+#include "log.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+static volatile uint16_t times_callback_called = 0;
```

Comment:
try removing `volatile` see if it breaks.

Reply:
still add it back in if it doesn't break :P

Reply:
removing `volatile` causes an infinite loop